packages/pricing/src/services/pricing-module.ts

```diff
@@ -85,7 +85,7 @@ export default class PricingModuleService<
 > implements PricingTypes.IPricingModuleService
 {
   protected baseRepository_: DAL.RepositoryService
-  protected readonly pricingRepository_: PricingRepositoryService
+  protected readonly pricingRepository_: RepositoryTypes.PricingRepositoryService
```

Comment:
todo: Since this is part of the DAL and could be changed in the future by the user, shouldn't this one be part of the types package?